tripgo.swagger.yaml

```diff
@@ -121,212 +131,180 @@ paths:
         200:
           description: Successful response
           schema:
-            type: object
-            description: Glogal information for this region.
-            properties:
-              regionCode:
-                type: string
-                description: Internal identifier for the region.
-              numberOfServices:
-                type: integer
-                description: Total number of services for this region.
-              realTime:
-                type: object
-                $ref: '#/definitions/RealtimeSnapshot'
-              transitModes:
-                type: array
-                description: Detailed mode information for this region.
-                items:
-                  type: object
-                  properties:
-                    identifier:
-                      type: string
-                      description: A specific public transit mode-identifier, e.g., `pt_pub_bus` or `pt_pub_gondola`.
-                    numberOfServices:
-                      type: integer
-                      description: Total number of services for this region's public transport mode.
-                    realTime:
-                      type: object
-                      $ref: '#/definitions/RealtimeSnapshot'
-                    operators:
-                      type: array
-                      description: TSP list for this mode and region.
-                      items:
+            type: array
+            items:
+              type: object
+              description: TSP information.
+              properties:
+                name:
+                  type: string
+                  description: TSP name.
+                mode:
+                  type: array
+                  items:
+                    description: Detailed information for this operator under a specific public transport mode.
+                    type: object
+                    properties:
+                      numberOfServices:
+                        type: integer
+                        description: Total number of services for this operator and mode.
+                      realTime:
                         type: object
-                        properties:
-                          name:
-                              type: string
-                              description: TSP's short name.
+                        $ref: '#/definitions/RealtimeSnapshot'
```

Comment:
For each endpoint we shall use a specific object ignoring alerts for agencies and routes as they are needed. We shall use allOf definitions.